diff --git a/node_modules/@ossiana/node-libcurl/dist/libcurl.d.ts b/node_modules/@ossiana/node-libcurl/dist/libcurl.d.ts
index ee69fe7..61f59db 100644
--- a/node_modules/@ossiana/node-libcurl/dist/libcurl.d.ts
+++ b/node_modules/@ossiana/node-libcurl/dist/libcurl.d.ts
@@ -38,7 +38,7 @@ export type LibCurlCookieAttrObject = {
     value: string;
 };
 export type LibCurlCookiesAttr = Map<string, LibCurlCookieAttrObject>;
-export type LibCurlHeadersAttr = Map<string, string>;
+export type LibCurlHeadersAttr = Headers;
 export type LibCurlInterfaceInfo = string;
 export type LibCurlJA3FingerPrintInfo = string;
 export declare enum LibCurlJA3TlsVersion {
diff --git a/node_modules/@ossiana/node-libcurl/dist/libcurl.js b/node_modules/@ossiana/node-libcurl/dist/libcurl.js
index 52e1231..9428cc0 100644
--- a/node_modules/@ossiana/node-libcurl/dist/libcurl.js
+++ b/node_modules/@ossiana/node-libcurl/dist/libcurl.js
@@ -261,17 +261,8 @@ class LibCurl {
     getResponseHeadersMap() {
         this.checkSending();
         const headers_ = this.m_libCurl_impl_.getResponseHeaders();
-        return headers_
-            .split("\r\n")
-            .slice(1)
-            .reduce((e, t) => {
-            if (!t) {
-                return e;
-            }
-            const [key, value] = t.split(": ");
-            e.set(key, value);
-            return e;
-        }, new Map());
+        const lines = headers_.split("\r\n").filter((header) => !header.startsWith('HTTP/') && !!header)
+        return new Headers(lines.map(line => line.split(': ')))
     }
     getResponseStatus() {
         this.checkSending();
@@ -382,11 +373,11 @@ class LibCurl {
             this.m_requestHeaderMap_.clear();
             return;
         }
-        if (!this.m_requestHeaderMap_.has("Accpet") &&
+        if (!this.m_requestHeaderMap_.has("Accept") &&
             !this.m_requestHeaderMap_.has("accept")) {
             this.m_requestHeaderMap_.set("Accept", "*/*");
         }
-        if (!this.m_requestHeaderMap_.has("Accpet-Encoding") &&
+        if (!this.m_requestHeaderMap_.has("Accept-Encoding") &&
             !this.m_requestHeaderMap_.has("accept-encoding")) {
             this.m_requestHeaderMap_.set("Accept-Encoding", "gzip, deflate, br, zstd");
         }
